native_client/README.md

```diff
@@ -66,6 +66,33 @@ cd ../DeepSpeech/native_client
 make deepspeech
 ```
 
+## Building with AOT model
+
+First, please note that this is still experimental. AOT model relies on TensorFlow's [AOT tfcompile](https://www.tensorflow.org/performance/xla/tfcompile) tooling. It takes a protocol buffer file graph as input, and produces a .so library that one can call from C++ code.
+To experiment, you will need to build TensorFlow from [github.com/mozilla/tensorflow master branch](https://github.com/mozilla/tensorflow). Follow TensorFlow's documentation for the configuration of your system.
+When building, you will have to add some extra parameter and targets.
+
+Bazel defines:
+* `--define=TF_HAS_NATIVE_MODEL=1`: to toggle AOT support.
+* `--define=DS_MODEL_TIMESTEPS=x`: to define how many timesteps you want to handle. Relying on prebuilt model implies we need to use a fixed value for how much audio value we want to use. Timesteps defines that value, and an audio file bigger than this will just be dealt with over several samples. This means there's a compromise between quality and minimum audio segment you want to handle.
+* `--define=DS_MODEL_FRAMESIZE=y`: to define your model framesize, this is the second component of your model's input layer shape. Can be extracted using `summarize-graph` TensorFlow's tool.
```

Comment:
"TensorFlow's `summarize_graph` tool", not the other way around. Note the underline.